src/App.js

```diff
@@ -1,24 +1,21 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React from "react";
+import logo from "./logo.svg";
+import "./App.css";
+
+import { BrowserRouter as Router, Route } from "react-router-dom";
+import { Provider } from "react-redux";
```

Comment:
Import all libraries at the top of the file, and then import your local files in the order of nesting from the farthest one